### ISSUE TITLE
Part of text missing: having an architecture which is flexible enough to support**not**

### ISSUE THREAD

**Issue by rutsky** [A/largescaleJS_ru#139]
This paragraph misses it's ending in english version (and has strange russian translation):

```
This is a relatively trivial decision in smaller codebases but for larger
applications, having an architecture which is flexible enough to support**not**
```

https://github.com/shuvalov-anton/largescaleJS_ru/blob/b213a4765965e2d6c71095b2afc45335f6fa32ad/_includes/translation/eng/03_think-long-term.md

What is the source of english version?

I found english version only in the author's blog post: http://addyosmani.com/largescalejavascript/:

```
This is a relatively trivial decision in smaller codebases but for larger
applications, having an architecture which is flexible enough to support
not caring about the libraries being used in your modules can be of
great benefit, both financially and from a time-saving perspective.
```

**Comment by A** [comment]
Good catch! I think it has been pandoc/readability parser's bug. PR is welcome :D

**Comment by rutsky** [comment]
So what is the proper source of the english version?

Should I copy/paste from Addy Osmani's blog (http://addyosmani.com/largescalejavascript/) or sources of his book can be found somewhere?

**Comment by A** [comment]
yes, it is http://addyosmani.com/largescalejavascript/

**Comment by A** [comment]
Time to update compiled books. link to #113

**Comment by rutsky** [comment]
@shuvalov-anton I believe this issue is resolved and can be closed sinece PR #142 is merged.

**Comment by A** [comment]
@rutsky  Yep, but we should update either of the ebooks as well.
